src/com/palmergames/bukkit/towny/command/TownCommand.java

```diff
@@ -1325,37 +1324,42 @@ private void townPlots(CommandSender sender, String[] args) {
 			return;
 		}
 
-		List<String> out = new ArrayList<>();
+		List<Component> out = new ArrayList<>();
 		out.add(ChatTools.formatTitle(town + " Town Plots"));
-		out.add(Colors.Green + "Town Size: " + Colors.LightGreen + town.getTownBlocks().size() + " / " + town.getMaxTownBlocksAsAString() 
-			+ (!town.hasUnlimitedClaims() 
-				? (TownySettings.isSellingBonusBlocks(town) 
-						? Colors.LightBlue + " [Bought: " + town.getPurchasedBlocks() + "/" + TownySettings.getMaxPurchasedBlocks(town) + "]" 
-						: "") 
-					+ (town.getBonusBlocks() > 0 
-						? Colors.LightBlue + " [Bonus: " + town.getBonusBlocks() + "]" 
-						: "") 
-					+ (TownySettings.getNationBonusBlocks(town) > 0 
-						? Colors.LightBlue + " [NationBonus: " + TownySettings.getNationBonusBlocks(town) + "]" 
-						: "")
-				: ""));
+		out.add(Component.text("Town Size: ", NamedTextColor.DARK_GREEN)
+			.append(Component.text(town.getTownBlocks().size() + " / " + town.getMaxTownBlocksAsAString(), NamedTextColor.GREEN))
+			.append(town.hasUnlimitedClaims() ?

```

Comment:
```suggestion
			.append(!town.hasUnlimitedClaims() ?

```

To match the code that is being replaced.